### ISSUE TITLE
'list' object not callable

### ISSUE THREAD

**Issue by dfreedman55** [django-import-export/django-import-export#354]
I'm new to Django so troubleshooting debugs/errors is a bit new to me, so my apologies in advance if I should know how to fix this.

I did the installation, configuration, edited admin.py to create my resource classes and I'm using 'ImportExportModelAdmin' for admin integration.  I'm able to see the Import/Export buttons on list view for my table (a table of Contacts, for example).  When I click on import, I get an error.  When I click on Export, I chose a file format, click submit and get an error.

The error in both cases is:  'list' object not callable.

Unsure how to proceed towards a fix.  Please advise.  Thanks in advance for any assistance.

**Comment by dfreedman55** [comment]
Here is what is in my admin.py file for my Contact table:

<pre>
class ContactResource(resources.ModelResource):
    class Meta:
        model = Contact
        skip_unchanged = True
        report_skipped = True
        fields = ('contact_name', 'company_name', 'location_name', 'role', 'phone1', 'phone2', 'email',)
</pre>

Since I'm getting a TypeError, I was thinking maybe it's because above the 'fields' variable is a tuple not a list.  I tried changing the ()'s to []'s with no improvement.  Below are the details of my TypeError, I can provide the line 195 traceback info if needed.

=========================================

TypeError at /admin/customers/contact/import/

'list' object is not callable

Request Method: 	GET
Request URL: 	http://127.0.0.1:8000/admin/customers/contact/import/
Django Version: 	1.7.9
Exception Type: 	TypeError
Exception Value: 	

'list' object is not callable

Exception Location: 	c:\code-projects\MYPYTHON\lib\site-packages\import_export\admin.py in import_action, line 195
Python Executable: 	c:\code-projects\MYPYTHON\Scripts\python.exe
Python Version: 	2.7.10
Python Path: 	

['c:\\code-projects\\MYPYTHON\\WEBDEV',
 'C:\\Windows\\system32\\python27.zip',
 'c:\\code-projects\\MYPYTHON\\DLLs',
 'c:\\code-projects\\MYPYTHON\\lib',
 'c:\\code-projects\\MYPYTHON\\lib\\plat-win',
 'c:\\code-projects\\MYPYTHON\\lib\\lib-tk',
 'c:\\code-projects\\MYPYTHON\\Scripts',
 'c:\\python27\\Lib',
 'c:\\python27\\DLLs',
 'c:\\python27\\Lib\\lib-tk',
 'c:\\code-projects\\MYPYTHON',
 'c:\\code-projects\\MYPYTHON\\lib\\site-packages']

Server time: 	Wed, 2 Dec 2015 11:23:58 -0800

**Comment by dfreedman55** [comment]
Disregard.  I figured out my problem.

In my 'class ContactAdmin', the call was:
<pre>resource_class = ['ContactResource']</pre>

When it should just have been:
<pre>resource_class = ContactResource</pre>

I knew this from the documentation but thought maybe it was an error because my Django webserver was crashing.  Then I realized it was my error, not the documentation (should have figured).  My ContactResource class came AFTER my ContactAdmin class in my admin.py file.  So basically, ContactAdmin was trying to call ContactResource class which had not been instantiated yet and that's what was causing my web server to crash with:

<pre>    
resource_class = ContactResource
NameError: name 'ContactResource' is not defined
</pre>

Sorry to have submitted this issue prematurely.  Closing out.
